spiffe-helper.go

```diff
@@ -209,6 +219,28 @@ func (s *sidecar) writeKey(file string, data []byte) error {
 	return ioutil.WriteFile(file, pem.EncodeToMemory(b), keyFileMode)
 }
 
+// writeKey takes a private key as a slice of bytes,
```

Comment:
amend the comment

Reply:
Changed the implementation of write key to accomodate append parameters for open file and changed the implementation to use write on *File.